src/components/NcModal/NcModal.vue

```diff
@@ -937,6 +937,10 @@ export default {
 	width: 100%;
 	height: 100%;
 
+	:deep(*) {
+		box-sizing: border-box;
+	}
+
```

Comment:
While I think it's good to have this globally, I somehow feel this is not the correct place to put this rule. It will affect all components, but it's located in `NcModal`. Don't we have a global CSS file or so?

Reply:
AFAIK, this rule was removed from global CSS styles and added to NcContent instead. This PR just aligns styles for all components, whether DOM elements are mounted inside NcContent or outside of it

Reply:
> While I think it's good to have this globally, I somehow feel this is not the correct place to put this rule. It will affect all components, but it's located in `NcModal`. Don't we have a global CSS file or so?

But it applies styles only to the content of `.modal-wrapper`, not truly globaly

Reply:
> > While I think it's good to have this globally, I somehow feel this is not the correct place to put this rule. It will affect all components, but it's located in `NcModal`. Don't we have a global CSS file or so?
> 
> But it applies styles only to the content of `.modal-wrapper`, not truly globally

Ah, yes, sorry. I missed that it is `scoped`. Fine then.